crawl-ref/source/describe-spells.cc

```diff
@@ -499,6 +499,15 @@ static string _effect_string(spell_type spell, const monster_info *mon_owner)
         }
         if (you.immune_to_hex(spell))
             return "(immune)";
+
+        if (spell == SPELL_MINDBURST
+            || spell == SPELL_PAIN)
+        {
+            const dice_def dam = _spell_damage(spell, hd);
+            return make_stringf("(%dd%dx%d%%)", dam.num, dam.size,
```

Comment:
This seems a bit confusing. "It does `1d15x20%` damage? So, 1d3 damage?"

I changed it to `20%,1d15` (in the example.) Probably still confusing, but maybe less so?

`20%->1d15` might also be reasonable. Open to more suggestions.